package.json

```diff
@@ -26,6 +26,7 @@
     "@angular/router": "16.0.6",
     "@nx/angular": "16.3.2",
     "firebase": "^9.22.2",
+    "lodash": "4.17.21",
```

Comment:
Is this library required?

Reply:
Resolved in https://github.com/usamazansari/expenses-tracker/pull/70/commits/759c97595047c22d34946c37b281c1942caad418